### PR TITLE
Draft tube model is created

### PR DESCRIPTION
**Pull request by pandeysudan27**
Two of the draft tube can be choosen, Conical diffuser and Moody spreading pipes. The svg figure in the documentation is not enlarged. I could not fixed it.

### CODE REVIEW

File: OpenHPL/Icons/DraftTube.mo [hunk 1]
```diff
@@ -0,0 +1,41 @@
+within OpenHPL.Icons;
+partial class DraftTube "Draft tube icon"
+  parameter Boolean vertical=false "Display vertical icon instead"
+    annotation(Dialog(group = "Icon"),
+    choices(checkBox = true));
```
Comment (dietmarw) [inline OpenHPL/Icons/DraftTube.mo]: What is the point of this unused parameter?

File: OpenHPL/Waterway/DraftTube.mo [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    p_o1 + p_o2 = o.p;
+  end if;
+  annotation (
+    Documentation(info="<html>
+<p>Two of the draft tubes are modeled using <i>Momentum balance . </i>They are:</p>
+<ul>
+<li><b>Conical diffuser:</b> It is the most well-know draft tube which has efficiency of around 90&percnt;  and mostly used for low head reaction turbines.</li>
+<li><b>Moody spreading draft tubes:</b> When conical diffuser length exceeds beyond its stability for high head reaction turbines, either a elbow type draft tube is used which has around 70&percnt; of efficiency. However, other choice is to use Moody spreading draft tube that has efficiency of around 80&percnt;. The construction and design of Moody spreading draft tube is daunting and time consuming but it is mostly choosen for handling water whril at turbine&apos;s outlet.</li>
+</ul>
+<p><br>The conical diffuser and Moody spreading draft tubes are shown below:</p>
```
Comment (dietmarw) [inline OpenHPL/Waterway/DraftTube.mo]:
```suggestion
<p>The conical diffuser and Moody spreading draft tubes are shown below:</p>
```

File: OpenHPL/Waterway/DraftTube.mo [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+<p><img src=\"modelica://OpenHPL/Resources/Images/conicalDiffuser.svg\"/></p>
```
Comment (dietmarw) [inline OpenHPL/Waterway/DraftTube.mo]:
```suggestion
<p><img src=\"modelica://OpenHPL/Resources/Images/conicalDiffuser.svg\" style=\"width:60%\"></p>
```

File: OpenHPL/Waterway/DraftTube.mo [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+<p><img src=\"modelica://OpenHPL/Resources/Images/moodySpreadingDT.svg\"/></p>
```
Comment (dietmarw) [inline OpenHPL/Waterway/DraftTube.mo]:
```suggestion
<p><img src=\"modelica://OpenHPL/Resources/Images/moodySpreadingDT.svg\" style=\"width:60%\"></p>
```

**Review by dietmarw — COMMENTED** [review]
I've left some comments and suggestions how to fix the scaling. You will only see this in OpenModelica since Dymola's HTML renderer does not support the svg scaling yet. Still I would not worry about that. Simply accept the suggestions by clicking on the "Commit suggestion" button.

File: OpenHPL/Icons/DraftTube.mo (hunk 1 above)
Comment (pandeysudan27) [reply OpenHPL/Icons/DraftTube.mo]: I do not know actually. I duplicated icon of pipe made by Luibomyr. 
(The mesh-grid coloring is also not supported in slanted icons like drafttube)

Comment (dietmarw) [reply OpenHPL/Icons/DraftTube.mo]: Well I know what it is for in the Icon you copied it from (I implemented that) but it does no sense here. So be careful when copying over code blindly. You should always know what the code does before using it :smirk:

Comment (pandeysudan27) [reply OpenHPL/Icons/DraftTube.mo]: Thanks. I hope to not repeat again.

**Comment by dietmarw** [comment]
Contained in #20

**Pull request closed without merging**